sites/kit.svelte.dev/src/routes/api/+server.js

```diff
@@ -0,0 +1,14 @@
+import { json } from '@sveltejs/kit';
+
+let x = 0;
+export async function GET({ url, setHeaders, request }) {
+	console.log('ENDPOINT', x++);
+
+	const search = url.searchParams.get('search') || '';
+
+	setHeaders({
+		'cache-control': 'max-age=60',
+		Vary: 'todos-cache'
+	});
+	return json({ a: Math.random() });
+}
```

Comment:
is this meant to be here or was it for debugging/testing?

Reply:
lol oops this is a leftover, I'll remove it